INSTETC/INSTETC-IOC-01App/Db/INSTETC.db

```diff
@@ -17,6 +17,16 @@ record (longin, "$(P)CS:MOT:STOP:ALL")
 	field(MDEL, -1)
 }
 
+## String field for the status of motor movement
+record(bi, "$(P)CS:MOT:MOVING:STR") 
+{
+	field(DESC, "IOCs with motion to complete")
+    field(INP, "$(P)CS:MOT:MOVING CP MS")
+	field(ZNAM, "STATIONARY")
+    field(ONAM, "MOVING")
+	info(archive, "VAL")
+}
```

Comment:
Do we need to archive this PV? It will always have the same value as CS:MOT:MOVING